core/src/main/java/org/bitcoinj/core/NetworkParameters.java

```diff
@@ -734,4 +735,8 @@ public int getBIP65Height() {
     public int getCoinType() {
         return coinType;
     }
+
+    public String [] getDefaultMasternodeList() {
```

Comment:
We can mark this method abstract and remove its body.

Reply:
I can do that, but must also add this "empty" method to all of NetworkParameters derived classes.

There is another way to handle this also, like other getter functions in this class.

Reply:
You're right, nevermind then.